### PR TITLE
Gpu flux readme

### PR DESCRIPTION
**Pull request by mauvais2**

### CODE REVIEW

File: run_flux.md
Comment (kmclough) [inline run_flux.md:1]: This is a good start at telling users how to run Flux, but it needs some context:

- Need a short summary at the beginning saying what Flux is (e.g., "Flux is a new resource management and scheduling system developed at LLNL. It has replaced SLURM as the native scheduler on LLNL clusters corona and tioga, and will eventually do so on other clusters. You need to learn basic Flux commands in order to run batch jobs and make use of GPU nodes on corona and tioga.")
- Need to link to this file from the main README.
-

File: run_tf_gpu.md
Comment (kmclough) [inline run_tf_gpu.md:1]: Again, this file needs some context:

- Why do we care about running TensorFlow on GPU? What models used by AMPL depend on TF? (Answer: only graphconv.) What is the advantage of using GPU? Do we need GPU to run the model, or only for training?
- Need to say these instructions are specific for LLNL clusters.
- These instructions are for clusters with Nvidia GPUs. What do users need to do on AMD clusters?

**Review by kmclough — CHANGES_REQUESTED** [review]

File: run_tf_gpu.md
Comment (kmclough) [reply run_tf_gpu.md:1]: Oh, and this file also needs to be linked from the main README file.

**Comment by mauvais2** [comment]
OBE by [Simple pip reqs](https://github.com/ATOMScience-org/AMPL/pull/235).

**Pull request closed without merging**